spec/models/user_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+describe User do
+
+  include TestFactories
+
+  before do
+    @user = authenticated_user
+    @post = associated_post
+  end
+
+  describe "#favorited(post)" do
+    it "returns 'nil' if the user has not favorited the post" do
+        expect( @favorite ).to eq(nil)
```

Comment:
How can this test be changed to call the method under test (as `describe`d on line 12)?